src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import React, { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const TabsPage: FC = React.memo(() => {
+  const { tabId } = useParams();
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(({ id, title }) => (
+            <li
+              data-cy="Tab"
+              className={classNames({ 'is-active': tabId === id })}
+              key={id}
+            >
+              <Link to={`../${id}`}>{title}</Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {tabs.find(({ id }) => (
```

Comment:
let't move this to the separate variable
```suggestion
        {... || ...
```